index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang = "en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Traffic & Conversion Summit</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'> <!-- Import Google Font -->
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="./src/css/styles.css">
+  </head>
+
+  <body>
+    <header>
+      <!--  Begin HEADER     -->
+      <div class="topBar">
+            <ul class="social_icons">
+                <li><a class="socialimg" href="https://www.facebook.com"><img src="./src/images/facebook.png" alt="Facebook icon"></a></li>
+                <li><a class="socialimg" href="https://www.twitter.com"><img src="./src/images/twitter.png" alt="Twitter icon"></a></li>
+                <li>English</li>
+            </ul>
+        </div>
+      <nav class="navbar">
+        <div class="mobileNavbar">
+          <div class="navbarItem1">
+               <a class="hamburguer" href="#"><img src="./src/images/hamburguerIcon.png" alt="hamburguer"></a>
+          </div>
+           <div class="navbarItem2">
+               <a class="logo" href="#"> <img src= "./src/images/logoBlack.jpg" alt"logo"></a>
+          </div>
+          <div class="mobileMenu">
+            <ul class="itemsMobileMenu">
+              <li class="close_menu"> <img src="./src/images/close_menu.png" alt="Close"/></li>
+              <li class="itemProgram"> <a href="#mainProgram">Program</a> </li>
+              <li class="itemSpeaker"> <a href="#speaker">Speaker</a> </li>
+              <li class="itemAbout"> <a href="about.html">About</a> </li>
+              <li class="itemAbout"> <a href="index.html">Home</a> </li>
+            </ul>
+            <ul class="itemsDesktopMenu">
+              <li class="itemProgram"> <a href="#mainProgram">Program</a> </li>
+              <li class="itemSpeaker"> <a href="#speaker">Speaker</a> </li>
+              <li class="itemAbout"> <a href="about.html">About</a> </li>
+              <li class="itemAbout"> <a href="index.html">Home</a> </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
```

Comment:
- [ ] As requested by the previous reviewer, please ensure that the header is centered to match the design guidelines. Try adding equal padding left and right or use a `width: 85%; min-width: 360px;` and `margin: auto;`.
### Your design
![3](https://user-images.githubusercontent.com/94551484/190767668-874792bd-e4b7-4661-b205-14147f69f3ac.png)
![31](https://user-images.githubusercontent.com/94551484/190767989-0bac0ab7-10fd-4e72-850d-4af18b305217.png)
### Design guidelines
![f3](https://user-images.githubusercontent.com/94551484/190770364-2acc90f8-7651-4c9a-8ac9-20471f1fe7fa.png)